test/fixtures/node_modules/invalid-pjson/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "invalid"
+}
```

Comment:
Should we still have the trailing new line here?

Reply:
Is there a rule somewhere in guidelines about trailing new lines?
Linter doesn't fail, so we could leave it as-is. I don't mind to add the trailing new line though.

Reply:
It's a general thing that will show up in the Github UI as a red marker and will often be auto-fixed by editors. Having no trailing new line is usually only done if there's a specific reason for it. Given that this file is called "invalid-pjson/package.json", it may direct people's attention to the wrong thing (the missing newline) instead of the thing that's actually invalid (no property value).